css/style/root.css

```diff
@@ -0,0 +1,172 @@
+/* Global */
+html, body {
+  height: 100%;
+}
+
+
+body {
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
```

Comment:
this `overflow: hidden` prevents scrolling from the touchpad in the OS simulator. A bit annoying, although keyboards arrows still work...